test/prog/sktable/LDA-PW91/Non-Relativistic/WS/config

```diff
@@ -0,0 +1 @@
+C C
```

Comment:
Missing line ending?

Reply:
Yes. I used the opportunity to fix it for all cases.